oauthenticator/oauth2.py

```diff
@@ -292,6 +292,14 @@ def _userdata_url_default(self):
     def _logout_redirect_url_default(self):
         return os.getenv("OAUTH_LOGOUT_REDIRECT_URL", "")
 
+    custom_403_message = Unicode(
+        config=True, help="""The message to be shown when user was not allowed"""
```

Comment:
Can we put the default message here as the traitlet's first arg (default) rather than use a method for this?

Also I think most installations do not use an allow-list explicitly. So perhaps the message can be more generic? "Sorry, you are not currently authorized to use this hub. Please contact the hub administrator"?

Reply:
Done in https://github.com/jupyterhub/oauthenticator/pull/484/commits/1d6c6baf4c0cf0d552314d46cf2727d788c78f5a. Thanks a lot for the review @yuvipanda ✨